index.html

```diff
@@ -607,31 +607,61 @@ <h3>Markup for indicating ruby annotation used as gikun or interlinear note</h3>
     <section>
       <h2>Alternatives to ruby</h2>
 
+      <p>[[?SSML]] and [[?PRONUNCIATION-LEXICON]] can be used for
+      providing phonemic/phonetic pronunciation of CJK ideograph
+      characters to speech synthesis engines.  They are not for visual
+      presentations but can control text-to-speech much better than
+      ruby.</p>
+      
       <section>
         <h3>SSML</h3>
         <p>
-          In EPUB publications, [[?SSML]] attributes can be attached to XHTML elements for specifying phonemic/phonetic pronunciations. 
-          [[?SSML]] attributes are used by text-to-speech engines for speech synthesis but they are not used for visual rendering. 
-          [[?SSML]] can control text-to-speech much better than ruby.
+	  [[?SSML]] uses symbol collections (such as IPA and JEITA
+	  IT-4006) to represent the sounds of human languages.
+	  Phonemic/phonetic pronunciation is represented by sequences
+	  of such symbols.
+        </p>  
+        <p>
+	  EPUB 3 allows SSML attributes to be used within XHTML content
+	  documents in EPUB publications.  In the upcoming version,
+	  EPUB 3.3, these attributes are moved to [[epub-tts-10]].
+	  Meanwhile, the W3C Accessible Platform Architectures Working
+	  Group is developing [[spoken-html]].  [[spoken-html]] describes
+	  two possible approaches for adding SSML attributes to HTML elements.
         </p>
         <p>
-          However, it has been reported that attaching [[?SSML]] to CJK ideographic characters thoroughly significantly 
-          increases the authoring cost. 
-          Although [[?SSML]] has been used by some textbook publishers in Japan, it is unlikely to be widely used for trade books.      
+	  [[?SSML]] is widely used for digital textbooks by more than
+          one textbook publisher in Japan.  Meanwhile, it has been
+          reported that attaching [[?SSML]] attributes to CJK
+          ideographic characters significantly increases the authoring
+          cost.  DAISY textbooks in Japan do not use SSML, since they
+          include recorded voice.  Trade books in Japan do not use
+          SSML either.
```

Comment:
```suggestion
          SSML neither.
```

Reply:
I think that "either" is correct here.  Grammarly agrees with me.

Reply:
Ahhh, yes. sorry for making noise here.